src/components/formProducts/FormProducts.js

```diff
@@ -22,44 +33,50 @@ const FormProducts = ({ items }) => {
     }
   }, [itemName, items]);
 
-  const resetInput = () => setItemName('');
+  const resetInput = () => {
+    setItemName('');
+    setIsFiltering(false);
+  };
 
   return (
     <>
-      <form>
-        <label htmlFor="filter">Filter items</label>
-        <div className="filter_item">
-          <input
-            id="filter"
-            className="inputField"
-            value={itemName}
-            type="text"
-            onChange={handleChange}
-            placeholder="Start typing a product..."
-          />
-          <button
-            type="button"
-            className="button_filteritem"
-            onClick={resetInput}
-          >
-            x
-          </button>
-        </div>
-      </form>
-      {itemsFiltered?.length > 0 ? (
-        itemsFiltered.map((item, index) => (
-          <ProductForList
-            key={`${index}-${item.name}`}
-            item={item}
-            token={token.current}
-          />
-        ))
-      ) : (
-        <p>
-          No results. There isn't any product with '{itemName}' as the name in
-          the database.
-        </p>
-      )}
+      <div className="list-header">
+        <h3 className="title">HELLO!</h3>
```

Comment:
Por temas de [accesibilidad](https://www.w3.org/WAI/tutorials/page-structure/headings/), es importante tener un "orden" en el uso e los títulos y subtítulos (headings).

Como el título `Hello!` es el único en toda la vista, se recomienda usar: `<h1>`

```suggestion
        <h1 className="title">HELLO!</h1>
```

Por favor, es posible su cambio. Lo agradecería. ¿Qué opinan ustedes?

Reply:
Claro que sí. Se acomoda a lo que vimos de accesibilidad!

Reply:
***c6231c109871a2ca93e66bfa807f4bb0e7f23644*** listo!